realsense2_camera/launch/rs_launch.py

```diff
@@ -35,9 +35,9 @@
                            {'name': 'rgb_camera.profile',           'default': '0,0,0', 'description': 'color image width'},
                            {'name': 'rgb_camera.enable_auto_exposure', 'default': 'true', 'description': 'enable/disable auto exposure for color image'},
                            {'name': 'enable_color',                 'default': 'true', 'description': 'enable color stream'},
+                           {'name': 'enable_infra',                 'default': 'false', 'description': 'enable infra0 stream'},
```

Comment:
Please add this parameter to rs_multi_camera_launch.py also.

Reply:
Anyway, _rs_multi_camera_launch.py_ takes the params from _rs_launch.py_. So, it is getting added automatically.